tools/remark-cli/LICENSE

```diff
@@ -1,22 +0,0 @@
-(The MIT License)
```

Comment:
Please could you update https://github.com/nodejs/node/blob/e86b319e3f9caa60502b0b0235c34ef0d7250c23/tools/license-builder.sh#L88?